pkgs/data/icons/banana-cursor/default.nix

```diff
@@ -0,0 +1,30 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+}:
+stdenv.mkDerivation rec {
+  pname = "banana-cursor";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "ful1e5";
+    repo = "banana-cursor";
+    rev = "v${version}";
+    sha256 = "sha256-PI7381xf/GctQTnfcE0W3M3z2kqbX4VexMf17C61hT8=";
+  };
+
+  dontBuild = true;
+  installPhase = ''
+    mkdir -p $out/share/icons
+    mv themes/Banana $out/share/icons
```

Comment:
this can be simplified with the `install` command

Reply:
How so?

Reply:
Yeah you right, sorry.